### ISSUE TITLE
Memory Clock OC Fail at any mhz

### ISSUE THREAD

**Issue by KatilTako77** [NebuTech/NBMiner#886]
My high quality hynix mem Rtx 3060ti Lhr card, which I have been using for 6 months at 3000 mhz, now gives an error even at 1000 mhz.I can't set any value of memclock . it keeps crahsing or start getting 19 mh and %0 fan(actually fans are working but hiveos cant see them). i use 510.60.02 driver and latest nbminer. i tried other miners , changed the riser and its cables, downgrade and upgrade the hiveos version, try with using another PSU at different rig.
![3123](https://user-images.githubusercontent.com/106037014/169717933-34bb5fc3-64c2-4da6-b14d-ecabba6b4981.PNG)

**Comment by cwkoomy** [comment]
1of my 3060ti always crash, now every miner test also become dag has been damaged, check overclocking settings. lower mem clock from 2550mem to 0mem to -500 to -1000, still crash. is my card vram faulty already ? before unlock 75% never face this problem. lolmiber DAG repair also fail... and reboot again & again

**Comment by KatilTako77** [comment]
> 1of my 3060ti always crash, now every miner test also become dag has been damaged, check overclocking settings. lower mem clock from 2550mem to 0mem to -500 to -1000, still crash. is my card vram faulty already ? before unlock 75% never face this problem. lolmiber DAG repair also fail... and reboot again & again

i still have this issue. I tried to benchmark Vrams on Occt app on windows but it had no errors. does your gpu have hynix mem too. I have some frineds which also have this issue. All of their mems are hynix. If you find a solution pls let me know.

**Comment by cwkoomy** [comment]
> > 1of my 3060ti always crash, now every miner test also become dag has been damaged, check overclocking settings. lower mem clock from 2550mem to 0mem to -500 to -1000, still crash. is my card vram faulty already ? before unlock 75% never face this problem. lolmiber DAG repair also fail... and reboot again & again
> 
> i still have this issue. I tried to benchmark Vrams on Occt app on windows but it had no errors. does your gpu have hynix mem too. I have some frineds which also have this issue. All of their mems are hynix. If you find a solution pls let me know.

yes 3060ti gaming oc pro rev3.0, hynix mem. another 1 rev2.0 hynix and other 4cards samsung without problem. only this card . what miner i try to use also error, 
gminer - dag has been damaged, check overclocking settings and error auto restart miner 0%fan
lolminer - dag verification detected then waiting repair and error reboot
bnminer - dag verifivation fail, card mark as red, cuda error unspecified launch failure err_no=4 then error reboot
t-rex - also forgot what cuda error reboot

i plug in windows run furmark , any gpu software test, score also nothing problem. gaming movie without screen error.
i dont know really is miner problem or this 100% unlock make my card faulty... if any solution please update me too. very bad no have any expert to give solution
